README.md

```diff
@@ -1 +1,7 @@
-# practice1
\ No newline at end of file
+# practice1
+
+def euclids(x,y):
+    if y == 0:
+        return x
+    else:
+        return euclids(y, x%y)
```

Comment:
I think you need some inputs to test this function

Reply:
Ok